docs/addons/fluxcd.md

```diff
@@ -29,6 +29,10 @@ const blueprint = blueprints.EksBlueprint.builder()
   .build(app, 'my-stack-name');
 ```
 
+## Secret Management for private Git repositories

```

Comment:
Have you tried it? I am not sure how exactly dependency is handled between aws provider, secret creation and flux installation.

Reply:
Yeah, i did try to create secret with external secrets to access a repo.